Added facilitator bios and map pins

aoby opened the pull request:
Note: the added function `formatted_date_with_start_and_end_times` in session.rb is also in https://github.com/code-dot-org/code-dot-org/pull/8499, along with tests.

cc @breville

breville commented on dashboard/app/views/pd/workshop_enrollment/new.html.haml:
```diff
@@ -3,88 +3,102 @@
+              %span.form-required-field
+                *
+            %div.controls
+              =form.text_field :district_name, class: 'span10'
```
you'll need to integrate the new district dropdown here..

breville commented on dashboard/app/views/pd/workshop_enrollment/new.html.haml:
```diff
@@ -3,88 +3,102 @@
+    .row-fluid
+      -# Left Column
+      .span6
+        .row-fluid
```
what prompted the move to fluid rows?

aoby replied:
I tried a number of things to get the 2 columns to work properly. I think the outer container is what did it. I can probably remove the -fluid parts.

aoby replied:
yeah, I switched it back to normal rows. Good catch.

breville commented on pegasus/sites.v3/code.org/public/js/workshop-search.js:
```diff
@@ -35,7 +42,7 @@ function loadWorkshops() {
         var infowindow_content = '';
 
-        if(typeof markers[hash]==='undefined'){
+        if (typeof markers[hash]==='undefined') {
```
need spaces around === ?

aoby replied:
I cleaned up this code the minimum needed to get it to pass the linting rules and function properly. It's not needed, but I prefer it as well so I'll add them. In general this code can be cleaned up quite a bit.

breville commented on pegasus/sites.v3/code.org/public/js/workshop-search.js:
```diff
@@ -48,15 +55,61 @@ function loadWorkshops() {
+        var infowindow_content = '';
+
+        if (typeof markers[hash]==='undefined') {
+          infowindow_content = compileHtmlNew(workshop, true);
```
should we camel case all these variables?  e.g. infoWindowContent?

aoby replied:
might as well. I copied and modified this function from above. I'll take another quick pass at cleaning it up slightly.

aoby commented:
I merged staging to integrate the district explorer, and updated. I also added css for .form-required-field to make the required *s red, which was somehow missing.

tanyaparker commented:
LGTM
